### ISSUE TITLE
Linux Client - Re-granting access on every startup

### ISSUE THREAD

**Issue by toxpal** [nextcloud/desktop#1655]
Using NC for years, but only via web and Android client. Yesterday, I installed NC Desktop Client for Linux, and while everything works, I experience a bug when closing NC desktop client requires re-granting access to account. This is exactly what happens step by step:

1. Client opens a popup, asking to launch browser - https://i.postimg.cc/m2tCz31J/a.png
2. Usual "Connect to your account" prompt - https://i.postimg.cc/Kv24Fnrt/b.png
3. Account access prompt - https://i.postimg.cc/3wMcbxHr/c.png

Then I see a success message that everything is OK and NC desktop client starts working. But if I close the client (or restart OS), the same procedure needs to be done again every time. 

I found another user reported the same bug, and it looks v2.6.1 RC1 fixed it for him (https://github.com/nextcloud/desktop/issues/1493). I'm using v2.6.1, but similar bug still exists on my system. This is NC client info:

```
Version 2.6.1git. For more information please click here.
This release was supplied by Nextcloud GmbH
Built from Git revision 5adbc0 on Nov 6 2019, 01:47:23 using Qt 5.13.2, OpenSSL 1.1.1d 10 Sep 2019
```

I'm running the latest version of Linux Manjaro KDE. My browser is set to remember my session (so I'm always logged in when opening NC via browser) and I also have NC Android client running at the same time.

If you need any more info, I'll be happy to provide it.

**Comment by misch7** [comment]
This looks like a duplicate of #1292

Please see here and try the AppImage: https://github.com/nextcloud/desktop/issues/1292#issuecomment-560008811

And please report your result then here :-)

**Comment by toxpal** [comment]
Thanks for super-fast reply.

Tried the AppImage, and I couldn't even login (everything happens as usually, popup, prompts to grant access, etc.), but after granting access and seeing a success message in browser, NC client stil claims I'm not logged in. Will try on another Manjaro KDE system tomorrow.

**Comment by misch7** [comment]
> Thanks for super-fast reply.

:-)

> ...but after granting access and seeing a success message in browser, NC client stil claims I'm not logged in.

Did you wait at least 30 seconds for the Client auth flow window to update? The UI will get improved later but that's work in progress. Since we recently (re-)increased the server poll interval from 5 to 30 secs, to free the servers from overhead, it's likely you have to be extra patient for the auth in the meantime ;-) 

> Will try on another Manjaro KDE system tomorrow.

👍

**Comment by toxpal** [comment]
No, I didn't But in 5.6.1, I didn't wait either. It starts working in a second or two. Thanks for the info, I will be more patient.

**Comment by misch7** [comment]
Closing here, fixed: https://github.com/nextcloud/desktop/issues/1292#issuecomment-560008811

**Comment by toxpal** [comment]
@misch7, the issue is **NOT** fixed and users confirmed the bug still exists - https://github.com/nextcloud/desktop/issues/1292#issuecomment-573589013

**Comment by claell** [comment]
@toxpal I reopened #1292. However, this seems to be a duplicate of that, is that correct? Then I will leave this closed and further tracking is done only in #1292.

**Comment by toxpal** [comment]
Yes, it' the same issue, so let's track #1292
